### PR TITLE
TWIR Week 29

### PR DESCRIPTION
**Pull request by cokiengchiara**
@giantswarm/sig-product FYI for this week releases - opened for AWS cc @sslavic (for AWS 9.3.4)

### CODE REVIEW

File: announcements/2020-Week-29/aws.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# :zap: This Week in AWS Releases (July 17, 2020):zap:
+
+[AWS v9.3.3](https://github.com/giantswarm/releases/tree/master/aws/v9.3.4) makes security fixes for 2 CVEs and updates managed apps to the latest releases.
```
Comment (sslavic) [inline announcements/2020-Week-29/aws.md:3]:
```suggestion
[AWS v9.3.4](https://github.com/giantswarm/releases/tree/master/aws/v9.3.4) makes security fixes for 2 CVEs and updates managed apps to the latest releases.
```

File: announcements/2020-Week-29/aws.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# :zap: This Week in AWS Releases (July 17, 2020):zap:
+
+[AWS v9.3.4](https://github.com/giantswarm/releases/tree/master/aws/v9.3.4) makes security fixes for 2 CVEs and updates managed apps to the latest releases.
+
+[AWS v9.0.7](https://github.com/giantswarm/releases/tree/master/aws/v9.0.7) updates managed apps to the latest releases.
+
+---
+
+Please help <!subteam^S0GSG846L> improve communication. **Was this helpful?** :thumbsup: :thumbsdown:
```
Comment (MartaD) [inline announcements/2020-Week-29/aws.md:9]:
```suggestion
Please let <!subteam^S0GSG846L> know if you have any feedback or questions by replying to this announcement in a thread.
```

File: announcements/2020-Week-29/kvm.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# :zap: This Week in KVM Releases (July 17, 2020):zap:
+
+
+Please let <!subteam^S0GSG846L> know if you have any feedback or questions by replying to this announcement in a thread.
```
Comment (MartaD) [inline announcements/2020-Week-29/kvm.md:4]:
```suggestion
Please reply to this announcement in a thread if you have any feedback or questions.
```
PO group handle doesn't look good, remove for now?

Comment (cokiengchiara) [reply announcements/2020-Week-29/kvm.md:5]: sure!

File: announcements/2020-Week-29/aws.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+[AWS v11.5.0](https://github.com/giantswarm/releases/tree/master/aws/v11.5.0) introduces _registry mirrors_ for Docker images, applies security fixes for three CVEs and contains more reliability improvements and fixes.
+
+[AWS v9.3.4](https://github.com/giantswarm/releases/tree/master/aws/v9.3.4) makes security fixes for 2 CVEs and updates managed apps to the latest releases.
+
+[AWS v9.0.7](https://github.com/giantswarm/releases/tree/master/aws/v9.0.7) updates managed apps to the latest releases.
+
+**Optional Apps Releases**  
+(Can be installed from App Catalogs)
+
+1. [Istio Operator](https://github.com/giantswarm/istio-operator) is now available as a Playground Catalog app and can be used to test and play with an Istio Service Mesh.
```
Comment (cokiengchiara) [inline announcements/2020-Week-29/aws.md:12]: @pipo02mix FYI

File: announcements/2020-Week-29/aws.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+2. [NGINX IC v1.7.3](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#173---2020-07-16) fixes a regression introduced in the previous version (Note: NGINX is Optional and not Pre-Installed in AWS 10+ clusters)
```
Comment (cokiengchiara) [inline announcements/2020-Week-29/aws.md:13]: @sslavic FYI

File: announcements/2020-Week-29/azure.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
+# :zap: This Week in Azure Releases (July 17, 2020):zap:
+
+**Optional Apps Releases**  
+(Can be installed from App Catalogs)
+
+1. [Istio Operator](https://github.com/giantswarm/istio-operator) is now available as a Playground Catalog app and can be used to test and play with an Istio Service Mesh.
```
Comment (cokiengchiara) [inline announcements/2020-Week-29/azure.md:6]:
```suggestion
[Istio Operator](https://github.com/giantswarm/istio-operator) is now available as a Playground Catalog app and can be used to test and play with an Istio Service Mesh.
```

File: announcements/2020-Week-29/azure.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
+# :zap: This Week in Azure Releases (July 17, 2020):zap:
+
+**Optional Apps Releases**  
+(Can be installed from App Catalogs)
+
+[Istio Operator](https://github.com/giantswarm/istio-operator) is now available as a Playground Catalog app and can be used to test and play with an Istio Service Mesh.
+2. [NGINX IC v1.7.3](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#173---2020-07-16) fixes a regression introduced in the previous version (Note: NGINX is Optional and not Pre-Installed in AWS 10+ clusters)
```
Comment (cokiengchiara) [inline announcements/2020-Week-29/azure.md:7]:
```suggestion
```

File: announcements/2020-Week-29/aws.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
```
Comment (cokiengchiara) [inline announcements/2020-Week-29/aws.md:13]:
```suggestion
2. [NGINX IC v1.7.3](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#173---2020-07-16) fixes a regression introduced in the previous version. (Note: NGINX is Optional and not Pre-Installed in AWS 10+ clusters)
```

File: announcements/2020-Week-29/kvm.md [hunk 9]
```diff
@@ -0,0 +1,12 @@
+# :zap: This Week in KVM Releases (July 17, 2020):zap:
+
+[KVM v12.0.0](https://github.com/giantswarm/releases/blob/master/kvm/v12.0.0) is the first Giant Swarm release which includes **Kubernetes v1.17**. Many core components and default apps have been updated for improved reliability and observability.
+
+**Optional Apps Releases**  
+(Can be installed from App Catalogs)
+
+1. [Istio Operator](https://github.com/giantswarm/istio-operator) is now available as a Playground Catalog app and can be used to test and play with an Istio Service Mesh.
```
Comment (cokiengchiara) [inline announcements/2020-Week-29/kvm.md:8]:
```suggestion
[Istio Operator](https://github.com/giantswarm/istio-operator) is now available as a Playground Catalog app and can be used to test and play with an Istio Service Mesh.
```

File: announcements/2020-Week-29/kvm.md [hunk 10]
```diff
@@ -0,0 +1,12 @@
+# :zap: This Week in KVM Releases (July 17, 2020):zap:
+
+[KVM v12.0.0](https://github.com/giantswarm/releases/blob/master/kvm/v12.0.0) is the first Giant Swarm release which includes **Kubernetes v1.17**. Many core components and default apps have been updated for improved reliability and observability.
+
+**Optional Apps Releases**  
+(Can be installed from App Catalogs)
+
+[Istio Operator](https://github.com/giantswarm/istio-operator) is now available as a Playground Catalog app and can be used to test and play with an Istio Service Mesh.
+2. [NGINX IC v1.7.3](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#173---2020-07-16) fixes a regression introduced in the previous version (Note: NGINX is Optional and not Pre-Installed in AWS 10+ clusters)
```
Comment (cokiengchiara) [inline announcements/2020-Week-29/kvm.md:9]:
```suggestion
```

**Pull request merged**